instat/static/InstatObject/R/data_object_R6.R

```
@@ -2406,6 +2406,24 @@ DataSheet$set("public","set_contrasts_of_factor", function(col_name, new_contras
 }
 )
 
+DataSheet$set("public", "make_quarters",  function(quater,start_month){
+     #qtr <- c()
+     mabb <- c("J", "F", "M", "A", "M", "J", "J", "A", "S", "O", "N", "D","J", "F", "M", "A", "M", "J", "J", "A", "S", "O", "N", "D")
+     if (quater == 1){
+       qtr <- paste(mabb[start_month:(start_month+2)],collapse="")
+     }
+     if (quater == 2){
+       qtr <- paste(mabb[(start_month+3):(start_month+5)],collapse="")
+     }
+     if (quater == 3){
+         qtr <- paste(mabb[(start_month+6):(start_month+8)],collapse="")
+       }
+     if (quater == 4){
+       qtr <- paste(mabb[(start_month+9):(start_month+11)],collapse="")
+     }

```

If quarter is not 1 to 4, then should we raise an error or return an error value?